webapp/src/Form/Type/ProblemType.php

```diff
@@ -81,6 +81,15 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             'label' => 'Use run script as compare script.',
             'required' => false,
         ]);
+        $builder->add('multipassProblem', CheckboxType::class, [
+            'label' => 'Multi-pass problem',
+            'required' => false,
+        ]);
+        $builder->add('multipassLimit', IntegerType::class, [
```

Comment:
We could hide this with some JavaScript but I will do this later

Reply:
Do you think we should check this value? 
For multi-pass problems, the passlimit must be >= 2, and for traditional problem it must be equal to 1. (I noticed that `getMultiPassLimit` checks for traditional problem, but the value stored in the db may be inconsistent)